platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -619,7 +619,8 @@ MGL_EXPORT
 
 /**
  Value to use for a text label. Feature properties are specified using tokens
- like {field_name}.
+ like {field_name}.  (Token replacement is only supported for literal text-field
+ values--not for data-driven property functions.)
```

Comment:
An override in style-spec-overrides-v8.json is probably inevitable, because the style specification doesn’t yet refer to “source functions”: mapbox/mapbox-gl-js#4154.